policy/goals/types.go

```diff
@@ -126,5 +126,6 @@ type (
 		TeamId       string
 		Organization string
 		Goal         Goal
+		Event        skill.EventIncoming
```

Comment:
Thinking about it, this feels a bit messy since this will only be set when the evaluation is triggered by the skills platform...

Reply:
We won't have this when we are coming from CLI for instance.

Reply:
Oh, right. yeah, that won't work in this case then.